<ui version="4.0">
 <class>WeatherDialog</class>
 <widget class="QDialog" name="WeatherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Погода</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_wind">
     <property name="text">
      <string>Ветер</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hbox_wind_speed">
     <item>
      <widget class="QLabel" name="label_speed">
       <property name="text">
        <string>Скорость (м/с):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spin_wind_speed"/>
     </item>
    </layout>
   </item>
   <!-- Добавьте все поля аналогично (QDoubleSpinBox для высоты волн, QSpinBox -->
   <item>
    <widget class="QPushButton" name="btn_save">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_cancel">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>